app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -33,47 +41,130 @@ fun ProfileScreen(
     userViewModel: UserViewModel = viewModel(factory = UserViewModel.Factory),
     innerPaddingValues: PaddingValues = PaddingValues(0.dp)
 ) {
+
+  val context = LocalContext.current
+
+  // Fake data extraction from the MVVM :
+  val alice =
+      User("uid_Alice", "Alice", "alice@gmail.com", 42, listOf("Z8qgzNOoQOR09x6QQYahcyzVXfE2"))
+
+  val bob = User("uid_Bob", "Bob", "bob@gmail.com", 64, emptyList())
+
+  val currentUser =
+      User("uid_current", "Current User", "user@gmail.com", 42424, listOf(alice.uid, bob.uid))
+
+  var friendList = listOf(alice, bob)
+  // friendList = emptyList()
```

Comment:
Suggestion: you can either add a val at the start of this commented line, or just remove it